src/main/java/com/asu/cse545/group12/Enumfile/RequestType.java

```diff
@@ -0,0 +1,16 @@
+package com.asu.cse545.group12.Enumfile;
+
+public enum RequestType {
+	
+	SIGNUP,
+    DEBIT_REQUEST,
+    CREDIT_REQUEST,
+    TRANSFER_REQUEST,
+    VIEW_TRANSACTIONS_REQUEST,
+    MODIFY_TRANSACTIONS_REQUEST,
+    VIEW_USERINFO_REQUEST,
+    UPDATE_USERINFO_REQUEST;
+    
```

Comment:
by default it will take 8 byte thing.If you think we need it.I will change
it.

On Thu, Oct 22, 2015 at 10:05 PM, Nagarjuna Myla <notifications@github.com>
wrote:

> In src/main/java/com/asu/cse545/group12/Enumfile/RequestType.java
> <https://github.com/chasethenag420/sbs/pull/98#discussion_r42833454>:
>
> > @@ -0,0 +1,16 @@
> > +package com.asu.cse545.group12.Enumfile;
> > +
> > +public enum RequestType {
> > +	
> > +	SIGNUP,
> > +    DEBIT_REQUEST,
> > +    CREDIT_REQUEST,
> > +    TRANSFER_REQUEST,
> > +    VIEW_TRANSACTIONS_REQUEST,
> > +    MODIFY_TRANSACTIONS_REQUEST,
> > +    VIEW_USERINFO_REQUEST,
> > +    UPDATE_USERINFO_REQUEST;
> > +
>
> DELETE_TRANSACTION_REQUEST
> DELETE_USER_REQUEST
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chasethenag420/sbs/pull/98/files#r42833454>.
>